### PR TITLE
[CH229] Feature - Need CIIA Agreement 

### PR DESCRIPTION
**Pull request by 92Sam**
Story details: https://app.clubhouse.io/masterventures/story/229

### CODE REVIEW

File: pages/profile.tsx [hunk 1]
```diff
@@ -26,14 +27,22 @@ const Profile: FC = () => {
...
 
       const xdvWallet = new Wallet({ isWeb: true });
 
-      await xdvWallet.open(accountName, values.passphrase);
+      const wallet = await xdvWallet.open(accountName, values.passphrase);
+
 
       const walletId = await xdvWallet.addWallet();
+      console.log(wallet)
+      console.log("new wallet id", walletId)
```
Comment (CoolHandLuke88) [inline pages/profile.tsx:45]: Please don't forget to remove console.logs when done with them.

File: pages/profile.tsx [hunk 2]
```diff
@@ -26,14 +27,22 @@ const Profile: FC = () => {
   const onSubmit = async (values: ProfileModel) => {
     try {
       const getCurrentWallet = global.localStorage.getItem(currentWallet);
+
+      // const didXdv = await getDidXdv(getCurrentWallet)
+      // const { profileData, provider} = didXdv
```
Comment (CoolHandLuke88) [inline pages/profile.tsx:32]: remove dead code

File: pages/profile.tsx [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { useDispatch, useSelector } from 'react-redux';
 import Head from 'next/head';
 import { Card } from 'reactstrap';
 import { Wallet } from 'xdv-universal-wallet-core';
+// import getDidXdv from '../utils/xdv-universal-wallet'
```
Comment (CoolHandLuke88) [inline pages/profile.tsx:6]: remove dead code

**Review by CoolHandLuke88 — APPROVED** [review]
@92Sam Few comments:

1. Please don't forget to remove `console.logs` when done with them.
2. Always remove dead code if it can be helped, we have Git history if we ever need to go back.

+1 sir

File: pages/profile.tsx (hunk 3 above)
Comment (92Sam) [reply pages/profile.tsx:6]: perfect working on that in the next PR bro

File: pages/profile.tsx (hunk 1 above)
Comment (92Sam) [reply pages/profile.tsx:45]: my bad fixing that in the new PR my friend.

**Comment by 92Sam** [comment]
@CoolHandLuke88 please my friend can ttake a look again. 

thanks.

**Pull request merged**